### ISSUE TITLE
3rd event

### ISSUE THREAD

**Issue by meeDamian** [lncm/ideas#10]
We have a date for the 2nd event, but it's never too early to ask for your preferred date for the 3rd event :). Leave comments below, or 👍 if someone already posted date that's best for you.

Next Bitcoin calendar event is on Oct 2nd (#FreeRoss day), but I don't think it makes sense to wait over 5 weeks for the next one.

PS. If you haven't added Bitcoin Calendar to your Google Calendar yet, you can do so [here](https://calendar.google.com/calendar/embed?src=meedamian.com_isk75e6iq93svn2rfh9sfr5tcc%40group.calendar.google.com&ctz=Asia%2FBangkok) (or, if you don't care about possible future updates, download in [iCal](https://calendar.google.com/calendar/ical/meedamian.com_isk75e6iq93svn2rfh9sfr5tcc%40group.calendar.google.com/public/basic.ics) format)

**Comment by meeDamian** [comment]
Should we have Google Calendar for our events too, or is putting them into `README.md` enough?

**Comment by meeDamian** [comment]
I had nothing, but issues with jekyll 😂. I have meedamian.com running on it, and I can't even touch it now, without causing a total calamity and making it completely broken 😬. 

**tl;dr:** I'd rather stay away from frontend. Ideally, as far as the laws of physics allow for.

**Comment by meeDamian** [comment]
https://github.com/lncm/ideas/pull/15

**Comment by meeDamian** [comment]
Settled on Sept 17th.
